frontend/src/Navbar/Navbar.tsx

```diff
@@ -21,13 +22,16 @@ const Navbar = () => {
         }
     }
 
+
     const renderBackground = (path: string) => {
         return ((path.length === 1 && location.pathname.length === 1) ||
-            (location.pathname.startsWith(path) && path.length === location.pathname.length))
+            (location.pathname.startsWith(path) && (path.length === location.pathname.length || (
+                path.length < location.pathname.length && location.pathname[path.length] === '/'
```

Comment:
this is really big OR -> if someone see this he doesn't know what happening here. Break it down into named conditions and use them